apis/rds/v1alpha1/generator-config.yaml

```diff
@@ -9,6 +9,16 @@ ignore:
     - CreateDBClusterInput.VpcSecurityGroupIds
     - ModifyDBClusterInput.VpcSecurityGroupIds
     - DBCluster.PendingModifiedValues
+    - DescribeDBInstancesInput.DBInstanceIndentifier
+    - CreateDBInstanceInput.DBInstanceIdentifier
+    - ModifyDBInstanceInput.DBInstanceIdentifier
+    - DeleteDBInstanceInput.DBInstanceIdentifier
+    - CreateDBInstanceInput.MasterUserPassword
+    - ModifyDBInstanceInput.MasterUserPassword
+    - CreateDBInstanceInput.VpcSecurityGroupIds
+    - ModifyDBInstanceInput.VpcSecurityGroupIds
+    - CreateDBInstanceInput.DBSecurityGroups
+    - ModifyDBInstanceInput.DBSecurityGroups
```

Comment:
```suggestion
```

I think we should be able to resolve to the generated fields, similar to `DomainIAMRoleName`

Reply:
There's a compiler error about type mismatching in the autogenerated code if I leave these as they are. I don't have the exact text but I added these after seeing that the DBCluster resource does the same thing for `VpcSecurityGroupIds`